src/blocks/KeccakTranscript.sol

```diff
@@ -1252,7 +1292,7 @@ library KeccakTranscriptLib {
         }
 
         // write byte indicating whether point is at infinity
-        if (Bn256.is_identity(point)) {
+        if (!Bn256.is_identity(point)) {
```

Comment:
😮

Reply:
Yeah - I was wondering too. The reason is that initial version of this `absorb` wasn't used actually while computation. Now with HyperKZG it is 100% used and debugging transcription revealed this mistake.